jbpm-designer-backend/src/test/java/org/jbpm/designer/web/server/CalledElementServletTest.java

```diff
@@ -84,4 +93,51 @@ public void testDoPostProfileAlreadySet() throws Exception {
                never()).findProfile(any(HttpServletRequest.class),
                                     anyString());
     }
+
+    @Test
+    public void testGetJavaTypeNames() throws Exception {
+        final PackageDataModelOracle oracle = mock(PackageDataModelOracle.class);
+        when(oracle.getProjectPackageNames()).thenReturn(Arrays.asList("org"));
+
+        ModelField[] modelFields = new ModelField[]{
+                new ModelField("this",
+                               "org.Address",
+                               ModelField.FIELD_CLASS_TYPE.REGULAR_CLASS,
+                               ModelField.FIELD_ORIGIN.SELF,
+                               FieldAccessorsAndMutators.ACCESSOR,
+                               DataType.TYPE_THIS),
+                new ModelField("street",
+                               String.class.getName(),
+                               ModelField.FIELD_CLASS_TYPE.REGULAR_CLASS,
+                               ModelField.FIELD_ORIGIN.DECLARED,
+                               FieldAccessorsAndMutators.BOTH,
+                               DataType.TYPE_STRING),
+                new ModelField("homeAddress",
+                               Boolean.class.getName(),
+                               ModelField.FIELD_CLASS_TYPE.REGULAR_CLASS,
+                               ModelField.FIELD_ORIGIN.DECLARED,
+                               FieldAccessorsAndMutators.BOTH,
+                               DataType.TYPE_BOOLEAN),
+                new ModelField("number",
+                               Integer.class.getName(),
+                               ModelField.FIELD_CLASS_TYPE.REGULAR_CLASS,
+                               ModelField.FIELD_ORIGIN.DECLARED,
+                               FieldAccessorsAndMutators.BOTH,
+                               DataType.TYPE_NUMERIC_INTEGER)};
+
+        Map<String, ModelField[]> fields = new java.util.HashMap<>();
+        fields.put("org.Address",
+                   modelFields);
```

Comment:
FYI This could be replaced by `Collections.singletonMap(...)`.

Reply:
@triceo -- same comment as in previous - thanks